src/server/components/lists/searches/translators-interpreters.ts

```diff
@@ -37,22 +41,33 @@ export const translatorsInterpretersQuestionsSequence = [
 ];
 
 const serviceTypeToNoun: Record<string, string> = {
-  translation: "translator",
-  interpretation: "interpreter"
+  translation: "Translators",
+  interpretation: "Interpreters",
 };
 
 function makeResultsTitle(country: string, servicesProvided: string[]): string {
-  const sanitisedServicesProvidedQuery = servicesProvided.map(service => serviceTypeToNoun[service]).filter(Boolean)
-  const interpretationOnly = sanitisedServicesProvidedQuery.includes(serviceTypeToNoun.interpretation) && sanitisedServicesProvidedQuery.length === 1;
-  const article = interpretationOnly ? "an" : "a";
-  return `Find ${article} ${sanitisedServicesProvidedQuery.join(" or ")} in ${country}`
+  const sanitisedServicesProvidedQuery = servicesProvided.map((service) => serviceTypeToNoun[service]).filter(Boolean);
+  return `${sanitisedServicesProvidedQuery.join(" or ")} in ${country}`;
 }
 
+interface SwornOutputTypes {
+  translators: boolean;
+  interpreters: boolean;
+  translatorsAndInterpreters?: boolean;
+}
+
+function hasSworn(results: TranslatorInterpreterListItemGetObject[]): SwornOutputTypes {
+  let swornOutput: SwornOutputTypes = {
+    translators: results.some((result) => result.jsonData.swornTranslations === "Yes"),
+    interpreters: results.some((result) => result.jsonData.swornInterpretations === "Yes"),
+  };
+
+  swornOutput = { ...swornOutput, translatorsAndInterpreters: swornOutput.translators && swornOutput.interpreters };
```

Comment:
I assume this is just a stylistic choice, but it's not necessary to replace swornOutput here. 

You can still add properties to `const`s -- consts just mean "don't redeclare this variable". 

```.ts
  const swornOutput: SwornOutputTypes = {
    translators: results.some((result) => result.jsonData.swornTranslations === "Yes"),
    interpreters: results.some((result) => result.jsonData.swornInterpretations === "Yes"),
  };
  swornOutput.translatorsInterpreters = swornOutput.translators && swornOutput.interpreters;
```

you can also use a getter function so you have access to `this`. you'll be able to return immediately as well

```.ts
  return {
    translators: results.some((result) => result.jsonData.swornTranslations === "Yes"),
    interpreters: results.some((result) => result.jsonData.swornInterpretations === "Yes"),
     get translatorsAndInterpreters () { return this.translators && this.interpreters  }
  };

```

Reply:
Ha, yeah I started out using a getter but the syntax started to get very verbose so I went with the other way. However, `translatorsAndInterpreters ` might not be necessary anymore if I update the .njk file that uses it 🤔

Reply:
Yeah I think just using and in the template will be cleaner if that’s the only usage